### ISSUE TITLE
When goal complete, remove edit and add CTA to create new goal

### ISSUE THREAD

**Issue by mbransn** [common-voice/common-voice#1960]
@Gregoor when a goal is complete the edit CTA should be gone and a black, pill shaped CTA should be at the bottom to 'Create New Goal'

<img width="433" alt="Screen Shot 2019-04-24 at 10 17 15 AM" src="https://user-images.githubusercontent.com/18686986/56679627-5fc06b80-667a-11e9-8c21-21b34efee84b.png">

cc @rleitan @viorelaioia

**Comment by Gregoor** [comment]
@mbransn what should happen when the goal type is `both`? Only show the create a new goal CTA when both are reached? And should it appear on both boxes?

**Comment by mbransn** [comment]
Great flags @Gregoor -- we missed success states in design so I've updated; take a look and attn: @viorelaioia @rleitan 

- [ ] If one goal is complete, a `both` goal can still be edited. And the CTA changes to 'View Award' On [Zeplin](https://zpl.io/aNmnlZe).

<img width="240" alt="dual-goal-1-complete" src="https://user-images.githubusercontent.com/18686986/56923196-cda3d300-6a7e-11e9-893a-b5368dfb5383.png">


- [ ] If both goals are complete the CTAs change to 'View Award' and the pencil icon is replaced with a 'Create a new goal' CTA at the bottom. On [Zeplin](https://zpl.io/Vx0vE5X).

<img width="240" alt="dual-goal-both" src="https://user-images.githubusercontent.com/18686986/56923202-d1cff080-6a7e-11e9-94fc-123f43207fc3.png">


- [ ] If a single (non-both) goal is complete the CTA changes to 'View Award' and the pencil icon is replaced with a 'Create new goal' CTA at the bottom. On [Zeplin](https://zpl.io/awJxvjn).

<img width="240" alt="single-goal-complete" src="https://user-images.githubusercontent.com/18686986/56923243-ec09ce80-6a7e-11e9-921a-642e8139e57b.png">

**Comment by Cronus1007** [comment]
@Gregoor  Are you still working on this issue?

**Comment by Gregoor** [comment]
@Cronus1007 I am not working on Common Voice anymore. But this looks up for grabs, @phirework is afaik still the person to talk to about getting PRs in, if yo uare interested in contributing.

**Comment by kshitij86** [comment]
@phirework @Gregoor Can I get assigned to work on this issue, as it is still open?

**Comment by Cronus1007** [comment]
@kshitij86  Feel free to send a PR.

**Comment by kshitij86** [comment]
Sure I'll send a PR fixing this soon!

**Comment by kshitij86** [comment]
@Cronus1007 I was able to setup `common-voice` on my machine, and I was even able to login but how do I populate the clips section so as to complete a goal? Could you please help me with this?

**Comment by phirework** [comment]
For recording sentences, you'll need to set `CV_IMPORT_SENTENCES` to true in your environment variables and restart the server to trigger db imports. For validating clips you'll need to actually record clips first.

**Comment by kshitij86** [comment]
@phirework I checked my Docker file and `CV_IMPORT_SENTENCES` is already `true` but when I try to record a sentence, it just says it's run out of sentences and sends me to the portal for collecting sentences.

**Comment by phirework** [comment]
Do you see in your docker logs that importing is happening? On server startup it should have messages like: `IMPORT -- importing ${locale_code}`. It's possible you're not getting to the English imports yet because it's still crunching through earlier sentences.

**Comment by Cronus1007** [comment]
@kshitij86  It would be a better approach if you join the `IRC Channel of Mozilla` and join the `common-voice` room and if you are facing issues while setting up the project locally then raise question there. :)

**Comment by kshitij86** [comment]
@Cronus1007 I'll try posting any further issues on Matrix I have, but I didn't get any feedback when I first posted there. I thought [Mozilla has moved away from `IRC`](https://wiki.mozilla.org/IRC)?

**Comment by kshitij86** [comment]
@phirework I see. I do have the following in my logs:

```
web        | [BE] IMPORT -- importing ab
web        | [FE] Built!
web        | [BE] IMPORT -- importing ace
web        | [BE] IMPORT -- importing af
web        | [BE] IMPORT -- importing ar
web        | [BE] IMPORT -- importing as
web        | [BE] IMPORT -- importing ast
web        | [BE] IMPORT -- importing ba
web        | [BE] IMPORT -- importing be
web        | [BE] IMPORT -- importing bg
web        | [BE] IMPORT -- importing bn
web        | [BE] IMPORT -- importing br
web        | [BE] IMPORT -- importing ca

```

So I should just wait till it gets to importing `en` sentences? Or can I fast track this somehow?

**Comment by phirework** [comment]
Yep, we're not on IRC anymore, Matrix is the official channel. Please keep in mind that last week was a holiday week for the US and many people who might've otherwise been able to help were offline with their families. 

You can fast-track it by deleting the other locale files in `server/data`. Or, for the purposes of testing locally, you can just try using one of the other languages to generate clips.

**Comment by Cronus1007** [comment]
@phirework @kshitij86  I am talking about this channel. https://chat.mozilla.org/#/room/#common-voice:mozilla.org. Indeed the whole mozilla community as per my observations was on a break the last week.

**Comment by moz-bozden** [comment]
Closing as stale...
